### DISCUSSION TITLE
Introduce "Custom styles / Templates" section in core cards for much easier change of CSS/styling of cards (like bubble card)

### DISCUSSION THREAD

**Discussion by nagtegaal** [home-assistant/frontend#22745] [Theming]
Introduce a "Styles" section in all the default available core cards to lower the bar for customizations and easier editing of CSS.
A magnificent example could be found in the amazing bubble-card.

This will eliminate the need for the excellent, but hard to grasp card_mod which is so widely used to style front-end in-card and card elements differently.

![](https://community-assets.home-assistant.io/original/4X/2/2/1/221b87fbe2808d97c5fda3cb280e6ba4fb0474e6.png)

**Comment by stefaanv** [comment]
Strong support for this request.
Many time you want to tweak a card just a little bit to f.i. increase text size, decrease overall size, etc
The only option I know of is **card_mod** but that can be really hard w.r.t. shadow DOM.
Also card_mod is a "fix" for something that should be a feature instead of some to be `fixed`
